packages/core/src/shared-wallets/add-shared-wallet/creation-flow/SharedWalletCreationStore.tsx

```diff
@@ -8,10 +8,14 @@ import React, {
   useMemo,
   useReducer,
 } from 'react';
-import { v1 as uuid } from 'uuid';
 import { makeInitialStateProvider } from '../../initial-state-provider';
 import { StateType, TransitionHandler, defineStateShape } from '../../state-utils';
 import { CoSigner, CoSignerDirty, CoSignerError } from './AddCoSigners';
+import {
+  createCosignerObject,
```

Comment:
could we please follow single approach, eg:
```suggestion
  createCoSignerObject,
```